pkgs/flarum/default.nix.bak

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, php, phpPackages, ... }:
```

Comment:
ignore this file for now, this is a mix of a `default.nix` we were handwriting plus trying to copy in some of the autogenerated nix code from `composer2nix`